views/partials/footer.hbs

```diff
@@ -0,0 +1,5 @@
+<footer>
+    <div class="wrapper">
+        <p>Copyright © 2011-2018 Sabka Bazaar Grocery Supplies Pvt Ltd</p>
```

Comment:
These labels should not be hard coded, use translations

Reply:
Done. Created a strings.js file that contains all the hard-coded strings stored as object properties.